### PR TITLE
docs(mongodb-compass): Improve MongoDB Compass page

### LINKED ISSUE #1477: [MongoDB Compass] SSL may need to be set to "Unvalidated (insecure)"
A customer reported that he received an error `self signed certificate in certificate chain` when attempting to use MongoDB Compass to open a DB tunnel and connect to his database. 

It eventually succeeded to connect by setting SSL to `Unvalidated (insecure)`.

On my side, I didn't need this parameter to make it work.

https://app.intercom.com/a/apps/w4oogu7s/inbox/inbox/all/conversations/12375700053331

### PR DESCRIPTION
**Pull request by brandon-welsch**
Fix #1477

### CODE REVIEW

File: src/_includes/mongo_db_tunnel.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{% warning %}
+If your MongoDB database use a Business plan, you have a replica set.
```
Comment (EtienneM) [inline src/_includes/mongo_db_tunnel.md:2]:
```suggestion
If your MongoDB database uses a Business plan, you have a replica set.
```

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 2]
```diff
@@ -1,72 +1,146 @@
...
-Internet. There are a couple of ways to get access to a Scalingo hosted
-database: building an SSH tunnel and making the database available on the
-internet. More information in the [dedicated page]({% post_url
-platform/databases/2000-01-01-access %}).
+Databases hosted on Scalingo are not by default directly available on Internet.
+There are a couple of ways to get access to a Scalingo MongoDB hosted database
+with MongoDB Compass:
+* Connection via an Encrypted Tunnel
+* Connection via the DB Tunnel of our CLI
+* Connection with TLS
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:19]: I would keep the link to the dedicated page (`platform/databases/2000-01-01-access`)

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 3]
```diff
@@ -1,72 +1,146 @@
...
+    * Connection String Scheme: mongodb
+    * Host: c393a9e3-42fe-4e33-9e6c-8ee815e9af88.my-app-7093.mongo.a.osc-fr1.
+    scalingo-dbs.com:31312 (Hostname + Port)
+* Authentication
+    * Authentication Method: Username / Password
+    * Authentication Mechanism: Default
+* TLS/SSL
+    * SSL/TLS Connection: On
+    * Certificate Authority (.pem): Depends on the region where your database
+    is hosted. You can find urls of our CA certificates at the end of the page.
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:55]:
```suggestion
    is hosted. You can find URLs of our CA certificates at the end of the page.
```

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 4]
```diff
@@ -1,72 +1,146 @@
...
+    and use it here
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: SSH with Identity File
+    * SSH Hostname/SSH Port/SSH Username: Depends on the region where your
+    database is hosted. You can find all information at the end of the page.
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:60]: > at the end of the page

Which page are you talking about?

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 5]
```diff
@@ -1,72 +1,146 @@
...
+    * SSH Identity File: Your private SSH key linked to the public SSH key you
+    uploaded on the [SSH Keys page](https://dashboard.scalingo.com/account/keys)
+    of the Scalingo dashboard.
+* Advanced
+    * Read Preference: Default
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:65]: I found the screenshot clearer than all this text with not that much information... Don't you think?

File: src/_posts/databases/mongodb/2000-01-01-compass.md [hunk 6]
```diff
@@ -1,72 +1,146 @@
...
+    * Authentication Method: Username / Password
+    * Authentication Mechanism: Default
+* TLS/SSL
+    * SSL/TLS Connection: On
+    * Certificate Authority (.pem): Download the right CA certificate below
+    and use it here
+* Proxy/SSH Tunnel
+    * SSH Tunnel/Proxy Method: None
+* Advanced
+    * Read Preference: Default
```
Comment (EtienneM) [inline src/_posts/databases/mongodb/2000-01-01-compass.md:127]: Once again, I don't understand the switch from screenshot to text, I find the screenshot clearer 🤔

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 5 above)
Comment (brandon-welsch) [reply src/_posts/databases/mongodb/2000-01-01-compass.md:65]: The screenshot is deprecated. Compass software doesn't look like the screenshots we currently have.

And the interface is divided into several tabs. So there would be too many screenshots to cover all the configuration for the different methods.

That's why I wrote it in text. It's much more maintainable and it allows to stay up to date despite the changes in the Compass interface

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 6 above)
Comment (brandon-welsch) [reply src/_posts/databases/mongodb/2000-01-01-compass.md:127]: answered above 😬

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 5 above)
Comment (EtienneM) [reply src/_posts/databases/mongodb/2000-01-01-compass.md:68]: mmmmmh I'm not completely convinced but OK. Can you remove the screenshots from the CDN?

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 4 above)
Comment (EtienneM) [reply src/_posts/databases/mongodb/2000-01-01-compass.md:63]: Can you add the name of the section where this information is readable?

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 5 above)
Comment (brandon-welsch) [reply src/_posts/databases/mongodb/2000-01-01-compass.md:69]: I don't want to have a lot of screenshots in the page to explain each case. The text is better here

>  Can you remove the screenshots from the CDN?

I will do when the PR is merged

File: src/_posts/databases/mongodb/2000-01-01-compass.md (hunk 4 above)
Comment (brandon-welsch) [reply src/_posts/databases/mongodb/2000-01-01-compass.md]: Is it better with a link ?

**Review by EtienneM — APPROVED** [review]

**Pull request merged**